source/manual/republishing-content.html.md

```diff
@@ -34,6 +38,19 @@ For example:
 You may wish to test first on Integration:
 [`publishing_api:bulk_republish:document_type[DocumentClass]`][republish-whitehall-document-type-jenkins-integration]
 
+For a short list of Content IDs, run:
+[`publishing_api:bulk_republish:republish_documents_by_content_ids[content_id_1 content_id_2]`][republish-whitehall-document-type-jenkins]
+
+For a significant number of Content IDs:
+Some preparation is needed for this as a CSV file needs to be in place. The CSV should have a column called content_id that contains all the relevant IDS. This should be added to the whitehall repository at:
+lib/tasks/{FILENAME}.csv
+[`publishing_api:bulk_republish:republish_documents_by_content_ids_from_csv[csv_file_name]`][republish-whitehall-document-type-jenkins]
+Ensure that the CSV is removed again after the job completes.
+
+To republish all documents:
+Proceed with caution, this is a lot of content and will take hours to complete.
```

Comment:
Might be worth elaborating on when you might want to use this, and also clarifying that 2nd line should be notified as the queues will be backed up, so they may see alerts.